packages/terra-consumer-nav/src/components/dangerous-html/DangerousHtml.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  text: PropTypes.string.isRequired,
+};
+
+const DangerousHtml = ({
```

Comment:
I Almost prefer this to be called SafeHTML... That follows closer to django and rails patterns. I get the whole call it dangerous so ppl don't use it thing.. but in this case this is HTML we deem safe to inject. FWIW, this could be a candidate for terra-core potentially? cc: @bjankord